### DISCUSSION TITLE
I hava NFO files，but how to use it in stash？

### DISCUSSION THREAD

**Discussion by fanglangxinghai** [stashapp/stash#5458] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I use [JavSP](https://github.com/Yuukiy/JavSP) to get NFO files and poster pictures，but how to use these in stash？Can anyone tell me how to solve this problem？

**Comment by feederbox826** [comment, marked as answer]
Stash does not support .nfo files, you can try using something like https://github.com/stashapp/CommunityScrapers/pull/1143 to import them
